vignettes/hazardItems.Rmd

```diff
@@ -31,13 +31,17 @@ This package **hazardItems** provides services in support of coastal change haza
 
 `thumb.service` expects an input json.url and returns a thumbnail png for the item.
 
+`refreshItemThumbnail` creates a new thumbnail PNG for the requested item ID and puts it to the user specified endpoint.
+
 `addLayer` service creates a new item from a zipped shapefile from an authenticated user.
 
 `template` service creates or replaces new child items based on an input id. 
 
-`setBaseURL` and `authenticateUser` are supporting functions for directing requests and posts to the proper development, testing or production tier, and checking proper access for posting data.
+`setBaseURL` is a supporting function for directing requests and posts to the proper development, testing or production tier 
 
+`authenticateUser` checks proper access for posting data to the currently specified tier/endpoint
```

Comment:
`authenticateUser` actually does the authentication. `checkAuth`, which is an internal function, is the thing that checks for valid auth.